packages/resourceful-redux/src/utils/cru-reducer-helper.js

```diff
@@ -40,8 +40,8 @@ export default function(state, action, {initialResourceMeta}, updatedMeta) {
     }
   }
 
-  // Without resources or labels, there is nothing to update
-  if (!hasResources && !label && !list) {
+  // Without resources, lists, or requests, there is nothing to update
```

Comment:
`// Without resources, a list, or a request name, there is nothing to update`

list/name are singular for actions